app/models/practice.rb

```diff
@@ -62,18 +62,6 @@ def reset_searchable_practices
     clear_searchable_cache if self.reset_searchable_cache
   end
 
-  def self.searchable_practices
```

Comment:
⚠️ generally, there were issues loading ActiveRecord objects from the cache without calling `.load` and for this one an error would throw so I removed it and opted to call it instead since we are only calling it once somewhere else